maxim/maxim.py

```
@@ -235,7 +289,13 @@ def __sync_prompt_chains(self):
             if self.raise_exceptions:
                 raise err
 
-    def __syncFolders(self):
+    def __sync_folders(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `__sync_folders`

Add return type annotation: `None`

(ANN202)